<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWidgets</class>
 <widget class="QWidget" name="EditorWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditorWidgets</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>266</width>
     <height>136</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>256</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>126</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8字节</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>浮点数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>双浮点数</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>文本</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>字节数组</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>35</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>16进制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>105</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>105</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>32</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>选择文本或者字节数组时，编辑框里显示的数据长度，并不是写入的长度</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
    <property name="placeholderText">
     <string>显示长度</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Ansi">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>80</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ansi</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_Unicode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unicode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_UTF8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UTF-8</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>60</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>要写入的文本编码：</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>165</y>
     <width>306</width>
     <height>96</height>
    </rect>
   </property>
   <widget class="QComboBox" name="comboBox_addr">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>166</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtCurrent</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>65</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>65</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_module">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>35</y>
      <width>296</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAtCurrent</enum>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>10</y>
     <width>176</width>
     <height>71</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>166</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>35</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
